autoscaler/controllers/gateway/config/root.go

```diff
@@ -38,7 +38,13 @@ func Calculate(dests *odigosv1.DestinationList, processors *odigosv1.ProcessorLi
 			return "", fmt.Errorf("no configer for %s", dest.Spec.Type)
 		}
 
-		configer.ModifyConfig(&dest, currentConfig)
+		err := configer.ModifyConfig(&dest, currentConfig)
+		if err != nil {
+			return "", err.Error()
```

Comment:
Prior, when there was an error in one destination, we would log an error but continue to the next destinations. If we return from the function, we will not have this behavior anymore.

I think what we should do in case of error, is to log it to the logger and update the status for the destination CR, and then continue the loop to the next destination